kokoro/windows/continuous.bat

```diff
@@ -1,4 +1,12 @@
 @echo on
+
+rem To speed up build, download and unpack an M2 repo cache.
+pushd %USERPROFILE%
+call gsutil -q cp "gs://ct4e-m2-repositories/m2-oxygen.tar" .
```

Comment:
You may be able to use `gsutil cat xxx | tar xf -` which should be more efficient.

Reply:
Nice.